<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Monitor</string>
    </property>
    <widget class="QLCDNumber" name="lcd_value">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sensor</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Temperatura</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcd_temp">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Modo: Medidor de pH</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>50</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Unidade: pH</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Estado: EstÃ¡vel</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>901</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Monitor AvanÃ§ado</string>
     </attribute>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>901</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>901</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Analyzer</string>
     </attribute>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>330</y>
        <width>901</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>721</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>MÃ©tricas</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Modo : 1000</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Unidade : 1000</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Maior Value : 1000</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>40</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Menor Value : 1000</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>70</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Maior Temperatura : 1000</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>90</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>MÃ©dia Temperatura : 1000</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>30</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>MÃ©dia Value : 1000</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>60</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Mediana Value : 1000</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>120</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Mediana Temperatura : 1000</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>901</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_inspect">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>20</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>false</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="text">
       <string>Inspecionar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_inspect_2">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>100</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>false</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="text">
       <string>Histograma aa</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>671</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Controles</string>
    </property>
    <widget class="QPushButton" name="btn_about">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>40</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>Sobre ðŸ‘ª</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_calib">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>Calibrar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_rec">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>Gravar ðŸ”´</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_hold">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>Hold âœ‹</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_config">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>150</y>
      <width>341</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>ConfiguraÃ§Ãµes</string>
    </property>
    <widget class="QSpinBox" name="sb_refresh_rate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
     <property name="value">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLabel" name="lb_refresh_rate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Taxa de Atualizacao (ms)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Modo de Funcionamento</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_mode">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>201</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_unit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>271</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Unidade de Medida</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_sensor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>[SELECIONE UM TIPO DE SENSOR]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>86501/551</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>86502/552</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>86503/553</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>86504/554</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>86505/555</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tipo de Sensor</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="ds_temp">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Temperatura</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chb_fixed_temp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Temperatura Fixa</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
